src/aggregator/aggregation/options.go

```diff
@@ -28,14 +28,19 @@ import (
 )
 
 var (
-	defaultHasExpensiveAggregations = false
+	defaultEnableExpensiveAggregations               = false
+	defaultEnableAggregationLastValueAdjustTimestamp = false
 )
 
 // Options is the options for aggregations.
 type Options struct {
-	// HasExpensiveAggregations means expensive (multiplication／division)
+	// EnableExpensiveAggregations means expensive (multiplication／division)
 	// aggregation types are enabled.
-	HasExpensiveAggregations bool
+	EnableExpensiveAggregations bool
+	// EnableAggregationLastValueAdjustTimestamp will enable last value
+	// aggregations to adjust the timestamp of the resulting last value
+	// aggregation.
+	EnableAggregationLastValueAdjustTimestamp bool
```

Comment:
May make more sense as `SnapLastAggregationTimestampToLast` or something? It's not really enable as force